gst/nnstreamer/tensor_repo.c

```diff
@@ -76,44 +85,96 @@ gst_tensor_repo_add_data (GstTensorData * data, guint myid)
  * @brief push GstBuffer into repo
  */
 gboolean
-gst_tensor_repo_push_buffer (guint nth, GstBuffer * buffer)
+gst_tensor_repo_set_buffer (guint nth, GstBuffer * buffer)
 {
   GST_TENSOR_REPO_LOCK (nth);
 
-  GstTensorData *data = gst_tensor_repo_get_tensor (nth);
+  GstTensorRepoData *data = gst_tensor_repo_get_repodata (nth);
   g_return_val_if_fail (data != NULL, FALSE);
 
   data->buffer = buffer;
-  GST_TENSOR_REPO_BROADCAST (nth);
+  if (DBG) {
+    unsigned long size = gst_buffer_get_size (data->buffer);
+    g_message ("Pushed [%d] (size : %lu)\n", nth, size);
```

Comment:
use GST_DEBUG/GST_INFO/... instead of g_message?

Reply:
Right. I will.